test/jdk/tools/jar/CreateJarBenchmark.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8276764
+ * @summary perform a jar creation benchmark
+ * @library /test/lib
+ * @modules jdk.jartool
+ * @build jdk.test.lib.Platform
+ *        jdk.test.lib.util.FileUtils
+ * @run testng CreateJarBenchmark
+ */
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.io.UncheckedIOException;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.spi.ToolProvider;
+import java.util.stream.Stream;
+import java.util.zip.ZipException;
+
+import jdk.test.lib.util.FileUtils;
+
+public class CreateJarBenchmark {
```

Comment:
This is a benchmark which should not be run as regression testing.   I wonder if we need this benchmark for future use.  If so I would suggest to convert it to JMH and place it under `test/micro/org/openjdk/bench/tools/jar` directory.   I'm okay with or without it in this patch.

Reply:
@LanceAndersen what's your opinion? I'm not totally sure of it's usefulness?

Reply:
My thought is to remove it from this PR, since we've already determined the change has little impact.
We can raise a new issue if we feel it's needed.

Reply:
The benchmark should use JMH.  @cl4es, what are your thoughts here?

Reply:
> My thought is to remove it from this PR, since we've already determined the change has little impact.
We can raise a new issue if we feel it's needed.

This is fine with me.

Reply:
I am Ok with separating this out as well but I think it would be nice to give Claes a chance to state his preference

Reply:
Removed benchmark